es-space-mgmt/main/default/lwc/reservationHelperForm/__tests__/reservationHelperForm.test.js

```diff
@@ -0,0 +1,125 @@
+            expect(customerDetailForm.sobjecttype).toBe(OBJECTTYPE);
+            expect(reservationDetailForm.state).toBe(STATE);
+        });
+    });
+
+    it('dispatch update customer event', () => {
+        const CUSTOMERID = '00Q5500000BFveDEAT';
+        const OBJECTTYPE = 'Lead';
+        const STATE = 'CA';
+        const STATE_NEW = 'NY';
+        const element = createElement('c-reservation-helper-form', {
+            is: ReservationHelperForm
+        });
+        element.customerid = CUSTOMERID;
+        element.objecttype = OBJECTTYPE;
+        element.state = STATE;
+        document.body.appendChild(element);
+        // Return a promise to wait for any asynchronous DOM updates. Jest
+        // will automatically wait for the Promise chain to complete before
+        // ending the test and fail the test if the promise rejects.
+        return Promise.resolve()
+            .then(() => {
+                const customerDetailForm = element.shadowRoot.querySelector(
+                    'c-customer-detail-form'
+                );
+                customerDetailForm.dispatchEvent(
+                    new CustomEvent('customerupdate', {
+                        detail: STATE_NEW
+                    })
+                );
+            })
+            .then(() => {
+                Promise.resolve().then(() => {
+                    expect(element.state).toBe(STATE_NEW);
+                });
+            });
+    });
+
+    it('dispatch draft reservation event', () => {
+        const CUSTOMERID = '00Q5500000BFveDEAT';
+        const OBJECTTYPE = 'Lead';
+        const STATE = 'CA';
+        //event Data
+        const EVENTDETAILS = {
+            startDate: '01/01/2021',
+            endDays: 30,
+            numberOfPeople: 400,
+            requestedMarket: 'San Mateo'
+        };
+        const element = createElement('c-reservation-helper-form', {
+            is: ReservationHelperForm
+        });
+        element.customerid = CUSTOMERID;
+        element.objecttype = OBJECTTYPE;
+        element.state = STATE;
+        document.body.appendChild(element);
+
+        // listen to flow navigate event
+        const handler = jest.fn();
+        element.addEventListener(FlowNavigationNextEventName, handler);
+        // Return a promise to wait for any asynchronous DOM updates. Jest
+        // will automatically wait for the Promise chain to complete before
+        // ending the test and fail the test if the promise rejects.
+        return Promise.resolve()
+            .then(() => {
+                const reservationDetailForm = element.shadowRoot.querySelector(
+                    'c-reservation-detail-form'
+                );
+                reservationDetailForm.dispatchEvent(
+                    new CustomEvent('draftreservation', {
+                        detail: EVENTDETAILS
+                    })
+                );
+            })
+            .then(() => {
+                Promise.resolve().then(() => {
```

Comment:
Extra `Promise.resolve()` is not needed

Reply:
done!

Reply:
https://github.com/trailheadapps/easy-spaces-lwc/commit/c30b02af8ee98863a264abe80992b9c4d9b9268a

Any idea why my recent commit does not show up in the PR? seems bizarre!

Reply:
ignore previous it showed after some time!